File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/bindings/ViMoTestFileNameBindingMyElementsRow.cpp
#include "ViMoTestFileNameBindingMyElementsRow.h"
#include <string>

namespace bindings
{
  std::string ViMoTestFileNameBindingMyElementsRow::getColumnCellLabelText()
  {
    return this->columnCellLabelText;
  }
  int ViMoTestFileNameBindingMyElementsRow::getRowIndex()
  {
    return this->RowIndex;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/bindings/ViMoTestFileNameBindingMyElementsRow.cs
/// <filename>
///     ViMoTestFileNameBindingMyElementsRow.cs
/// </filename>
namespace bindings
{
    public class ViMoTestFileNameBindingMyElementsRow
    {
        private string columnCellLabelText;

        public virtual string getColumnCellLabelText()
        {
            return this.columnCellLabelText;
        }

        private int RowIndex;

        public virtual int getRowIndex()
        {
            return this.RowIndex;
        }
    }
}
